CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
+
+  It's now possible to infer parameter and result types from the body of a multi-statement
```

Comment:
I thought it was just the result types that were inferred?

Reply:
It's both actually, just like single-statement closures do since the body participates in the type-check.